Automattic/wp-calypso
# Plugin Details: Show Marketplace Reviews

Fixes https://github.com/Automattic/dotcom-forge/issues/4563

## Proposed Changes

* Add the MarketplaceReviewsList component
* Link to the number of reviews link

| Top of the page  | Reviews area |
| ------------- | ------------- |
|![CleanShot 2023-11-29 at 16 31 38@2x](https://github.com/Automattic/wp-calypso/assets/5039531/b7854d71-ac32-4b39-b38b-f5b0a819d0d9)|![CleanShot 2023-11-29 at 16 31 31@2x](https://github.com/Automattic/wp-calypso/assets/5039531/c26db05c-dded-4fdb-a8ce-6d7b885a7af9)|

## Testing Instructions
* Go to a plugin page with reviews. Ex: `plugins/woocommerce-bookings`
* On the top of the page the number of the reviews should be displayed, click on it.
* You should be redirected to the bottom of the page and check if the reviews are displayed there
* Also test on the mobile view
* Also check plugins without a reviews. Ex: `/plugins/wordpress-seo-premium`. The review list component shouldn't be shown at all.

## Pre-merge Checklist
